tooling/cli/templates/tauri.conf.json

```
@@ -19,7 +19,8 @@
       }
     ],
     "security": {
-      "csp": null
+      "csp": null,
+      "capabilities": ["default-plugins"]
```

I don't like the `default-plugins` identifier, could we change that to just `default`?
```suggestion
      "capabilities": ["default"]
```
or should we inline the default capability here, so users used to allowlist, see something familiar? and we could try to find a way to list default plugins in the schema so they have auto-completion

---

let's make this change when we change the config schema to be stored in the project itself (so we can modify it)